### PR TITLE
created a search bar

### PR DESCRIPTION
**Pull request by Henrynunez112**
**Description**
A text box inserted to be used a search bar

### CODE REVIEW

File: src/cico/CicoPage.tsx [hunk 1]
```diff
@@ -42,9 +42,13 @@ function CicoPage(props: I18nProps & Props) {
...
   return (
     <div css={containerCss}>
-      <div></div>
+
+      <div>
+        <input placeholder="search" type="text" onChange={(e) => setSearch(e.target.value)} />
+      </div>
       {Object.keys(byCountries)
         .sort()
+        .filter((title) => title.toLowerCase().includes(search))
```
Comment (aaronmgdr) [inline src/cico/CicoPage.tsx:51]: tip: Filter and then sort. its more efficient :)

Comment (aaronmgdr) [reply src/cico/CicoPage.tsx:51]: Also sense on every keystroke its gonna be rerendering I would suggest creating a pure Component  for what goes on in the `map` part eg lines 54 to 100

Comment (Henrynunez112) [reply src/cico/CicoPage.tsx:51]: Thank you!!

Comment (aaronmgdr) [reply src/cico/CicoPage.tsx:51]: The other thing you will want to do is debounce the setSearch function. (do a search in repo, it should have an eample)

**Pull request merged**